client/src/modules/shopping-list/index.jsx

```diff
@@ -144,15 +143,13 @@ class ShoppingList extends Component {
           />
         )}
         {showUpdateForm && (
-          <ModalBox onCancel={this.hideUpdateForm}>
-            <CreationForm
-              defaultDescription={description}
-              defaultName={name}
-              label="Edit list"
-              onSubmit={this.updateListHandler(listId)}
-              type="modal"
-            />
-          </ModalBox>
+          <ModalForm
```

Comment:
consider my previous proposal: pass updateFormVisibility as a e.g. isVisible prop to ModalForm and decide there if creationForm should be render or not